### ISSUE TITLE
Using  `LegendRvHorizontal` with `MatplotFigureTwinx` crashes the app when double-clicked on the legend

### ISSUE THREAD

**Issue by kubapilch** [mp-007/kivy_matplotlib_widget#17]
I am trying to use the interactive legend with twinx graph. I got it to work even though the code is a bit different from a standard graph widget. After getting it to display correctly and react to a single click to hide/show each line, DoubleClick however crashes the app with a following error: 

```
File "/home/pi/.cache/pypoetry/virtualenvs/arfi-5TTrv0KZ-py3.9/lib/python3.9/site-packages/kivy_matplotlib_widget/uix/legend_widget.py", line 105, in on_touch_up
     self.cg_double_tap(touch, x, y)
   File "/home/pi/.cache/pypoetry/virtualenvs/arfi-5TTrv0KZ-py3.9/lib/python3.9/site-packages/kivy_matplotlib_widget/uix/legend_widget.py", line 520, in cg_double_tap
     self.legend_rv.doubletap(self.row_index)
   File "/home/pi/.cache/pypoetry/virtualenvs/arfi-5TTrv0KZ-py3.9/lib/python3.9/site-packages/kivy_matplotlib_widget/uix/legend_widget.py", line 445, in doubletap
     for idx,line in enumerate(self.figure_wgt.axes.lines): 
 AttributeError: 'NoneType' object has no attribute 'lines'
```

The error comes from the fact that `axes` is not set as I am not sure what I am supposed to set it to, with only one Y axis there is no confusion but what should be done in case there are two.

My code:
```python
mygraph = GraphGenerator(path, stack=stack)

self.graph_widget = MatplotFigureTwinx()

self.graph_widget.figure = mygraph.fig
self.lines = mygraph.fig.axes[0].get_lines() + mygraph.fig.axes[1].get_lines()
self.graph_widget.register_lines(self.lines)

legend = LegendRvHorizontal()
legend.figure_wgt = self.graph_widget
legend.set_data(self.graph_widget.lines)
```

For a quick fix that seems to work I have created a dummy `axes` and set the lines property, that seems to work fine.
```python
class DummyAxes():
    lines = []
```
And then:
```python
self.graph_widget.axes = DummyAxes()
self.graph_widget.axes.lines = self.graph_widget.lines
```

**Comment by mp-007** [comment]
thanks for reporting this bug. I have only test all legend widgets and tools for 1 axis figure. I will changed this when I have time.

**Comment by mp-007** [comment]
@kubapilch can you check if my last commit solve your issue 703f279f912d393d3846892cb6966574badda844

**Comment by kubapilch** [comment]
@mp-007 I have experienced some bugs when homing the twinx graph where the second axis lines would just move above the first axis lines and out of the screen. 

That made me just switch back to standard widget with less functional but more stable two axes. My current work schedule does not include reworking the graph widget again, but I can try to find some time for it.

**Comment by mp-007** [comment]
> @mp-007 I have experienced some bugs when homing the twinx graph where the second axis lines would just move above the first axis lines and out of the screen.
> 
> That made me just switch back to standard widget with less functional but more stable two axes. My current work schedule does not include reworking the graph widget again, but I can try to find some time for it.

I have made some fix for this in this commit 33679e2011c5634d9faed2aca9e604f7667c10e8. I

**Comment by mp-007** [comment]
fix with commit https://github.com/mp-007/kivy_matplotlib_widget/commit/703f279f912d393d3846892cb6966574badda844
